pkg/ring/model.go

```diff
@@ -45,18 +45,18 @@ func (d *Desc) AddIngester(id, addr string, tokens []uint32, state IngesterState
 		Addr:      addr,
 		Timestamp: time.Now().Unix(),
 		State:     state,
+		Tokens:    tokens,
```

Comment:
Created an issue for this just to not miss out: https://github.com/cortexproject/cortex/issues/1877

Reply:
I've added comment to AddIngester about tokens being replaced and test to make sure it will not break.